Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```
@@ -2248,6 +2248,13 @@
                             "$ref": "#/components/schemas/TopicImage"
                         }
                     },
+                    "custom": {
```

I think `custom` is a reasonable option here but the way things are written in `RenderMetadata.swift`, this will be encoded as `customMetadata`.

```suggestion
                    "customMetadata": {
```

---

done